src/Newsletter.php

```diff
@@ -199,6 +200,23 @@ public function updateContent(string $campaignId, string $html, array $options =
         return $response;
     }
 
+    public function sendCampaign(string $campaignId)
+    {
+        $response = $this->mailChimp->post("campaigns/{$campaignId}/actions/send");
+
+        $lastResponse = json_decode(($this->mailChimp->getLastResponse()['body']));
+
+        if ($lastResponse->status !== 200) {
+            throw CampaignException::withMessage($lastResponse->detail);
```

Comment:
Pass the entire `$lastResponse` to the exception and reach for `->detail` inside the exception.